backend/src/main/cards/test_cards.py

```diff
@@ -0,0 +1,29 @@
+from backend.src.main.cards.card import Card
+from backend.src.main.cards.cards import Cards
+
+
+def test_cards_starts_empty():
+    assert Cards().cards.__len__() == 0
```

Comment:
change this line to be more pythonic 
`assert not Cards().cards` 

See: https://www.python.org/dev/peps/pep-0008/
relevant block:
```For sequences, (strings, lists, tuples), use the fact that empty sequences are false.

Yes: if not seq:
     if seq:

No:  if len(seq):
     if not len(seq):```

Reply:
There are a few instances that this is not resolved